money/money.py

```diff
@@ -29,6 +29,18 @@ def currency(self) -> Currency:
 
         return self._currency
 
+    @classmethod
+    def from_sub_units(cls, sub_units: int, currency: Currency=Currency.USD):
+        """Creates a Money instance from sub-units."""
+        sub_units_per_unit = CurrencyHelper.sub_unit_for_currency(currency)
+        return cls(Decimal(sub_units) / Decimal(sub_units_per_unit), currency)
+
+    @property
+    def sub_units(self) -> int:
+        """Converts the amount to sub-units"""
+        sub_units_per_unit = CurrencyHelper.sub_unit_for_currency(self.currency)
+        return int(self._round(self.amount, self.currency) * sub_units_per_unit)
```

Comment:
why is _round needed here? i think it should have already been rounded in the constructor.

Reply:
Good point. I have updated the code to remove the unnecessary rounding.